recipes/sktools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sktools" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftbplus/skprogs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e63e6a38f920bbb85f1f8fdff2da5ec195a3f2ded354f6f09cf68cf69785b59d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install sktools/ -vv --no-deps
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - python >=3.6
+
+test:
+  imports:
+    - {{ name }}
+    - {{ name }}.hsd
+    - {{ name }}.calculators
+    - {{ name }}.skgen
```

Comment:
Are you planning to change the package name? Don't need templates for constant values.

Reply:
Package name is likely to stay, if preferred I can add the explicit name here.